components/ws-manager-api/core.proto

```diff
@@ -124,6 +127,29 @@ enum StopWorkspacePolicy {
     ABORT = 2;
 }
 
+enum StopWorkspaceReason {
+    // The user stopped the workspace
+    USER_STOPPED = 0;
+
+    // The user deleted the workspace
+    USER_DELETED = 1;
```

Comment:
How is a STOP different to DELETE? I would've assumed that on the ws-manager API level they are one and the same, no?

Reply:
Yes, but the user intent is different:
- Stopped because user wanted it stopped
- Stopped automatically because user wants it deleted

Reply:
Sure, but that's the API that Server/Dashboard exposes, not the API that `ws-manager` exposes. And here, we're dealing with the ws-manager API. 

A delete is effectively a Stop followed by a Workspace Delete. To a ws-manager, it doesn't really make sense to broaden it's domain to a delete when it currently doesn't understand what a delete is (from it's API surface perspective)